natgeo/mortar
# [SYTO-822] Create and Style The Latest Component

https://jira.natgeo.com/browse/SYTO-822

This story creates "The Latest" component, which is rendered by a component-data modeled (https://jira.natgeo.com/browse/AEM-5800) as "ContentPackage". The "ContentPackage" component will also contain the "Most Read" component. That work is slated for a future sprint. I imagine during the work for https://jira.natgeo.com/browse/SYTO-885 and https://jira.natgeo.com/browse/SYTO-886 there will be some refactoring of the instantiation of these components.

It should also be noted that the styling of the individual story cards is being handled in different stories as well.

To test:

- Pull feature/SYTO-822
- Run `npm test` to verify no errors
- Run `gulp serve`
- Load http://localhost:3000/contentPackages/contentPackage/
- Verify 4 cards are rendered
- Verify heading value can be updated
- Don't worry about layout of story cards yet
